looker_deployer/commands/deploy_content.py

```diff
@@ -125,8 +133,56 @@ def import_content(content_type, content_json, space_id, env, ini, debug=False):
         win_exec = ["cmd.exe", "/c"]
         gzr_command = win_exec + gzr_command
 
+    is_new_dash = "false"
+    existing_dash_alerts = []
+    existing_elements_w_alerts = []
+
+    if content_type == "dashboard":  ## only run for dashboards, looks can't have alerts
```

Comment:
Is it possible to move this into a separate function that is called from here to allow for unit tests?